samples/CMakeLists.txt

```diff
@@ -82,6 +83,8 @@ add_subdirectory(cpp)
 add_subdirectory(dnn)
 # FIXIT: can't use cvconfig.h in samples: add_subdirectory(gpu)
 
+add_subdirectory(java/tutorial_code)
+
```

Comment:
Please remove this. It doesn't work well for samples standalone build (build samples only against OpenCV installation or build directory).
Anyway ANT_EXECUTABLE/OpenCV_BINARY_DIR are not defined for these builds, and location of opencv.jar is not valid (winpack puts "jar" file under "java" subdirectory, not "bin").